cinema/serializers.py

```diff
@@ -125,10 +127,11 @@
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
-            ValidationError
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"]
+            # attrs["movie_session"].cinema_hall,
+            # ValidationError,
```

Comment:
These lines are commented out but seem to be part of the validation logic. If `cinema_hall` and `ValidationError` are required for validation, uncomment and integrate them properly.